meta-hailo/recipes-core/images/hailo-image-ci.bb

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: Copyright 2024 Siemens AG
+# SPDX-License-Identifier: MIT
+inherit image
+
+DESCRIPTION = "CI test image for HAILO"
+
+IMAGE_INSTALL += " \
+    hailo-pci-${KERNEL_NAME} \
+    hailo-firmware \
+    hailortcli \
+    python3-hailort \
+"
```

Comment:
Why a separate CI image? Did the build time exceeded the limit on github?

You want to explain such details, at least in the commit message (reasoning, not feature-listing).

Reply:
same as above https://github.com/siemens/meta-iot2050/pull/569#discussion_r1868978203

Reply:
That unfortunately makes no sense yet.

Reply:
ACK, I'll drop this and refine the commit message